src/main/java/edu/ucsb/cs156/example/entities/Roster.java

```diff
@@ -0,0 +1,30 @@
+package edu.ucsb.cs156.example.entities;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
```

Comment:
Do we need a separate database object for "Roster"?  

Or can we just attach the list of roster students directly to the course?

I think a course will only ever have one roster; I can't think of a use case for multiple rosters (and it does introduce some extra complexity).

Reply:
Factored out into separate issue #9